Resources/views/Collector/db.html.twig

```diff
@@ -373,24 +362,30 @@
             }
         }
 
-        {% if profiler_markup_version == 1 %}
-            function toggleRunnableQuery(target) {
```

Comment:
Please don't break the differences between the v1 and v2 profilers. We want to support Symfony 2.8 properly, and ``toggleRunnableQuery`` is necessary only on older versions